File: api/WebAPI/Models/Shipment.cs
using System;
using System.Collections.Generic;

namespace WebAPI.Models;

public partial class Shipment
{
    public int ShipmentId { get; set; }

    public int OrderId { get; set; }

    public string Origin { get; set; } = null!;

    public string Destination { get; set; } = null!;

    public bool HasArrived { get; set; }
}

File: api/express/src/model/shipment.ts
import {Entity, PrimaryGeneratedColumn, Column, Timestamp} from "typeorm";

@Entity({ name: "shipments" })
export class Shipment {

    @PrimaryGeneratedColumn({name: 'shipment_id'})
    shipmentId: number;

    @Column({name: 'order_id'})
    orderId: number;

    @Column()
    origin: string;

    @Column()
    destination: string;

    @Column({name: 'has_arrived'})
    hasArrived: boolean;
}